translations/mk/try_ruby_10.md

```diff
@@ -0,0 +1,25 @@
+---
+lang:   MK
+title:  Имаш 30 минути? Започни со програмирање во Руби веднаш!
+answer: 
+class:  stretcher chapmark
+ok:     
+error:  
+---
+
+Руби (анг. **Ruby**) е програмски јазик од Јапонија кој го револуционизира софтверскиот развој.
```

Comment:
I would actually not translate the name. I also checked the other translations, they use Ruby as well. 

> Ruby е програмски јазик од Јапонија кој го револуционизира софтверскиот развој.